packages/docs/content/components/button.mdx

```diff
@@ -472,6 +485,45 @@ For forward movement to the next page or step.
   </tbody>
 </table>
 
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+<table class="ds-c-table ds-c-table--compact ds-c-md-table--stacked" role="table" width="100%">
+  <thead role="rowgroup">
+    <tr role="row">
+      <th class="ds-u-fill--success-lightest" role="columnheader" scope="col">
+        Use
+      </th>
+      <th class="ds-u-fill--error-lightest" role="columnheader" scope="col">
+        Don't use
+      </th>
+    </tr>
+  </thead>
+  <tbody role="rowgroup">
+    <tr role="row">
+      <td
+        role="cell"
+        data-title="Document title"
+        width="50%"
+      >
+       Next
```

Comment:
It's funny that Medicare and Healthcare are opposite on this.

Reply:
I was just commenting. Nothing needs to change here

Reply:
I thought the same thing and brought that up in this video https://cmsgov.slack.com/archives/CT169N26L/p1669839659896879?thread_ts=1669821327.215169&cid=CT169N26L